src/model/error.messages.ts

```diff
@@ -0,0 +1 @@
+export const REDIS_PROBLEM: string = "There is a problem with redis: ";
```

Comment:
Since there's only one entry in here (for now) wonder if this could just be defined in the 'redis.store.ts' file, or is it referenced elsewhere too?